cluster-http/src/main/resources/reference.conf

```
@@ -26,6 +26,8 @@ akka.management {
 akka.management {
   health-checks {
     readiness-checks {
+      # Default HHealth chheck for cluster. Overwri the setting to replace it with
```

typo HH and Overwri

---

```suggestion
      # Default health check for cluster. Overwrite the setting to replace it with
```

My `h` key is acting up (see `chheck` too).